homeassistant/components/androidtv/diagnostics.py

```diff
@@ -20,7 +20,7 @@
 
 async def async_get_config_entry_diagnostics(
     hass: HomeAssistant, entry: ConfigEntry
-) -> dict[str, dict[str, Any]]:
+) -> dict[str, Any]:
```

Comment:
This would unlock #85525
The other possibility is to adjust diagnostics type hints to return `Mapping[str, Any]`